app/presenters/advanced_search_result_set_presenter.rb

```diff
@@ -1,10 +1,9 @@
 class AdvancedSearchResultSetPresenter < ResultSetPresenter
   include AdvancedSearchParams
 
-  def to_hash
+  def facet_tags_content
     super
       .merge(applied_filters: selected_filter_descriptions)
-        .except(:atom_url)
```

Comment:
Why is this `except` now gone?

Reply:
In a later commit I removal the whole method, since this stuff didn't seem to have an effect. The class that this inherits from does the same thing in `to_hash`: `applied_filters: selected_filter_descriptions`, and I've removed `to_hash` from both.